docs/configuration/good-practices.rst

```diff
@@ -221,3 +205,60 @@ Scheduled down time
 ===================
 
 * Change Backoff setting in application configuration
+
+
+
+
+About sharding
+==============
+
+`Sharding <https://en.wikipedia.org/wiki/Shard_%28database_architecture%29>`_ is
+horizontal scaling, where the data is partitioned in different *shards*.
+
+A client is then automatically assigned a particular shard, depending for
+example:
+
+* on the request authorization headers
+* on the bucket or collection id
+
+It is currently not possible to setup the sharding directly from the kinto
+settings, however it is already possible to set it up manually. [#]_
+
+.. [#] http://www.craigkerstiens.com/2012/11/30/sharding-your-database/
+
+
+At the HTTP level
+-----------------
+
+It is possible to handle the sharding at the HTTP level. For instance, using
+a third-party service that assigns a node to a particular user.
+
+This has the advantage to be very flexible: new instances can be added and
+this service is in charge of partitioning, downside being maintaining a new
+service for it.
```

Comment:
I removed it because I think it just adds some pointers to a mozilla specific tool. We can add it as an example, though. Updating the docs accordingly.